bindings/python/iota_sdk/client/_high_level_api.py

```diff
@@ -73,16 +125,30 @@ def find_blocks(self, block_ids: List[HexStr]) -> List[Block]:
 
     def retry(self, block_id: HexStr) -> List[HexStr | Block]:
         """Retries (promotes or reattaches) a block for provided block id. Block should only be
-           retried only if they are valid and haven't been confirmed for a while.
+        retried only if they are valid and haven't been confirmed for a while.
+
+        Args:
+            block_id: A block id.
+
+        Returns:
+            A list of `HexStr`s or `Block`s.
         """
         result = self._call_method('retry', {'blockId': block_id})
         result[1] = Block.from_dict(result[1])
         return result
 
     def retry_until_included(self, block_id: HexStr, interval: Optional[int] = None, max_attempts: Optional[int] = None) -> List[List[HexStr | Block]]:
         """Retries (promotes or reattaches) a block for provided block id until it's included (referenced by a
-           milestone). Default interval is 5 seconds and max attempts is 40. Returns the included block at first
-           position and additional reattached blocks.
+        milestone). Default interval is 5 seconds and max attempts is 40. Returns the included block at first
+        position and additional reattached blocks.
+
+        Args:
+            block_id: A block id.
+            interval: A retry interval in seconds. Defaults to 5.
+            max_attempts: A maximum number of retries. Defaults to 40.
+
+        Returns:
+            A list of lists of `HexStr`s or `Block`s.
```

Comment:
And here it's a List of Lists where the first element is the block id and the second one the block

Reply:
Fixed